packages/jest-snapshot/src/inline_snapshots.ts

```diff
@@ -60,56 +53,66 @@ const saveSnapshotsForFile = (
   snapshots: Array<InlineSnapshot>,
   sourceFilePath: Config.Path,
   prettier: any,
-  babelTraverse: Function,
+  babelTraverse: BabelTraverse,
 ) => {
   const sourceFile = fs.readFileSync(sourceFilePath, 'utf8');
 
-  // Resolve project configuration.
-  // For older versions of Prettier, do not load configuration.
-  const config = prettier.resolveConfig
-    ? prettier.resolveConfig.sync(sourceFilePath, {
-        editorconfig: true,
-      })
-    : null;
-
-  // Detect the parser for the test file.
-  // For older versions of Prettier, fallback to a simple parser detection.
-  const inferredParser = prettier.getFileInfo
-    ? prettier.getFileInfo.sync(sourceFilePath).inferredParser
-    : (config && config.parser) || simpleDetectParser(sourceFilePath);
+  // TypeScript projects may not have a babel config; make sure they can be parsed anyway.
+  const plugins = [];
+  if (/\.tsx?$/.test(sourceFilePath)) {
+    plugins.push([
+      require.resolve('@babel/plugin-syntax-typescript'),
```

Comment:
will this work in PnP? We can add a dependency on it I think

Reply:
Oops, had added it but apparently got rid of it at accidentally at some point and only kept the preset deps

Reply:
Should we add the node syntax plugin as well?

Reply:
Thought about that as well. If someone is using current node `transform: {}` I suppose it could be necessary?

Reply:
The only downside I can see is in the case that current-node-syntax will include a parser variant (e.g. of pipeline operator) that is incompatible/conflicts with a legacy variant declared in the user Babel config. In the `babel-jest` case that's not too bad because you can choose to ignore `babel-preset-jest`, but here you have no choice. But it's very unlikely to happen I guess.